docs/src/nodes/detailed-hydropower.md

```diff
@@ -1,406 +1,496 @@
+The [`HydroGenerator`](@ref) node represents a hydropower unit used to generate electricity in a hydropower system.
+In its simplest form, the [`HydroGenerator`](@ref) can convert potential energy stored in the reservoirs to electricity by discharging water between reservoirs at different head levels under the assumption that the reservoirs have constant head level.
+The conversion to electric energy can be described by an power-discharge relationship referred to as the PQ-curve.
+
+#### [Standard fields](@id nodes-hydro_generator-fields-stand)
 
-- **`id`**:
+[`HydroGenerator`](@ref) nodes build on the [`HydroUnit`](@ref EnergyModelsRenewableProducers.HydroUnit) and the  [`RefNetworkNode` ](@extref EnergyModelsBase.RefNetworkNode) nodes, but add additional fields.
+The standard fields are:
+
+- **`id`**:\
   The field `id` is only used for providing a name to the node.
-- **`cap::TimeProfile`**:
-  Can refer to either the installed power or discharge capactiy of the hydropower unit. 
-- **`opex_var::TimeProfile`**:
-  The variable operational expenses are based on the capacity utilization through the variable [`cap_use`](@ref man-opt_var-cap).
-- **`opex_fixed::TimeProfile`**:
-  The fixed operating expenses are relative to the installed capacity (through the field `cap`) and the chosen duration of a strategic period as outlined on *[Utilize `TimeStruct`](@ref how_to-utilize_TS)*.\
+  This is similar to the approach utilized in `EnergyModelsBase`.
+- **`cap::TimeProfile`**:\
+  The installed capacity corresponds to the nominal capacity of the node.
+  It can refer to either the installed power or discharge capacity of the hydropower unit.
+- **`opex_var::TimeProfile`**:\
+  The variable operational expenses are based on the capacity utilization through the variable [`cap_use`](@extref EnergyModelsBase man-opt_var-cap).
+- **`opex_fixed::TimeProfile`**:\
+  The fixed operating expenses are relative to the installed capacity (through the field `cap`) and the chosen duration of a strategic period as outlined on *[Utilize `TimeStruct`](@extref EnergyModelsBase how_to-utilize_TS)*.\
   It is important to note that you can only use `FixedProfile` or `StrategicProfile` for the fixed OPEX, but not `RepresentativeProfile` or `OperationalProfile`.
   In addition, all values have to be non-negative.
-- **`data::Vector{Data}`**:
-  An entry for providing additional data to the model. 
+- **`data::Vector{Data}`**:\
+  An entry for providing additional data to the model.
 
-!!! note "Additional constraints"
-    The `data` field can be used to add minimum, maximum and schedule constraints on discharge and power using the general constraints types described [here](@ref nodes-hydro-phil-con).
+  !!! note "Additional constraints"
+      The `data` field can be used to add minimum, maximum, and schedule constraints on the power generation using the general *[constraints types](@ref nodes-det_hydro_power-phil-con)*.
```

Comment:
You can put a constraint on power and/or discharge by setting the resource type you want to limit when defining the constraint.

Reply:
Fair enough, will look at it in the next commit.